tests/pkg/cache/cspccache/cache.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2020 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cspccache
+
+import (
+	"github.com/openebs/api/pkg/apis/types"
+	"github.com/openebs/cstor-operators/tests/pkg/infra"
+	"github.com/openebs/cstor-operators/tests/pkg/k8sclient"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
+)
+
+type CSPCResourceCache struct {
+	NodeList []string
+	Nodes    map[string]map[string]string
```

Comment:
Can we name it as NodeLabels/Something relevant to that? Since when I looking at spec builder I need to come over `CSPCResourceCache` to understand `Nodes` field.